project_definition/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Term Definitions in projects",
+    "version": "8.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "",
+    "summary": "",
+    "depends": [
+        'project',
+        'knowledge',
+        'document_page_partner_id',
```

Comment:
currently useless dependency, will need it if we want specific partner_id and project_ids associations in future. Module lives on same repo.